test/test_css_parser_loading.rb

```diff
@@ -128,6 +128,25 @@ def test_following_badly_escaped_import_rules
     end
   end
 
+  def test_loading_malformed_content_strings
+    file_name = File.dirname(__FILE__) + '/fixtures/import-malformed.css'
+    @cp.load_file!(file_name)
+    @cp.each_selector do |sel, dec, spec|
+      assert_nil dec =~ /wellformed/
+    end
+  end
+
+  def test_loading_malformed_css_brackets
+    file_name = File.dirname(__FILE__) + '/fixtures/import-malformed.css'
+    @cp.load_file!(file_name)
+    selector_count = 0
```

Comment:
I'm not a huge fan of the way I do this, as I wish there was a way I could just get an array of selectors. Right now, there are methods for each_selector and each_rule_set, but sometimes I just want to get the array of either and not go into a block. There was ways to output the string of selectors (rules_by_media_query(:all)), but then I'd have to split them again and parse, and I was worried I wouldn't parse correctly. This seems to do the job though.